ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml

```diff
@@ -250,6 +250,37 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity
+- as: azure-ipi-disc-oidc-arm-f14
+  cron: 26 13 12,27 * *
+  steps:
+    cluster_profile: azure-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
```

Comment:
For arm arch, I think need the following setting together:
```
      COMPUTE_NODE_TYPE: Standard_D4ps_v5
      OCP_ARCH: arm64
```

Reply:
thank you.